DevelopGitHub/GitHubExtension/GitHubExtension.Activity.External.WebAPI/Models/EventPayloads/EventPayloadConverter.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using GitHubExtension.Activity.External.WebAPI.Exceptions;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace GitHubExtension.Activity.External.WebAPI.Models.EventPayloads
+{
+    public class EventModelConverter: JsonConverter
```

Comment:
Please add Json in the name of the class due to converting json object not xml or html.